src/Docker/Client/Types.hs

```diff
@@ -1422,4 +1423,3 @@ toJsonKeyVal vs getKey getVal = JSON.Object $ foldl f HM.empty vs
 -- | Helper function that return an empty dictionary "{}"
 emptyJsonObject :: JSON.Value
 emptyJsonObject = JSON.Object HM.empty
-
```

Comment:
I'm mostly using hlint as a linter and stylish-haskell for some basic auto formatting. But I don't really have a process set up or anything like that. (I do have vim plugins that I use that invoke the above tools). 

Admittedly these are 2 different things. Hlint will offer suggestion not merely based on style but on utilizing better code practices. Ie. shortening code and what not. I'm not yet sure how to automate that with Travis. Stylish-haskell on the other hand is just formatting. However, hindent seems to be more powerful in that regard. 

And I was looking into using hindent in full swing so I don't have to make decisions about styling at all: http://chrisdone.com/posts/hindent-5 It's been updated to support only one style (as opposed to the previous version). There are some things that I object to but I guess I could learn to live with them for the sake of consistency. 

So in summary I guess what I want is commit a hindent and hlint file into the repo and make sure people use that. And have travis tests somehow enforce that.